frontend/src/widgets/RegionalGoalDashboard/GoalsPercentage.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from '@trussworks/react-uswds';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faUser,
+} from '@fortawesome/free-solid-svg-icons';
+import withWidgetData from '../withWidgetData';
+import colors from '../../colors';
+
+export function GoalsPercentageWidget({ data, loading }) {
+  if (loading) return (<div>Loading...</div>);
```

Comment:
This loading seems diff than the other widgets not sure if we want to change it to match.

Reply:
![image](https://user-images.githubusercontent.com/84350609/221990795-6fa155dc-ef88-4838-b032-c7b775d21506.png)

Reply:
fixed: [63230f8](https://github.com/HHS/Head-Start-TTADP/pull/1373/commits/63230f820c392066faeac68a75dbcf76fbe44271)